src/main/java/uk/gov/pay/webhooks/webhook/WebhookService.java

```diff
@@ -79,5 +86,19 @@ public WebhookEntity update(String externalId, String serviceId, List<JsonPatchR
             return webhookEntity;
         }).orElseThrow(NotFoundException::new);
     }
-    
+
+    public List<WebhookEntity> getWebhooksSubscribedToEvent(InternalEvent event) {
+        return list(event.live(), event.serviceId())
+                .stream()
+                .filter(webhook -> webhookHasSubscriptionForEvent(webhook, event))
+                .toList();
+    }
+
+    private boolean webhookHasSubscriptionForEvent(WebhookEntity webhook, InternalEvent event) {
+        return webhook.getSubscriptions().stream()
+                .map(EventTypeEntity::getName)
+                .map(uk.gov.pay.webhooks.webhookevent.EventMapper::getInternalEventNameFor)
```

Comment:
Can we use an `import` rather than including the fully-qualified name here?

Reply:
👍